tests/infra/ccf.py

```diff
@@ -261,45 +267,81 @@ def _add_node(self, node, lib_name, args, should_wait=True):
         )
 
         # If the network is opening, node are trusted without consortium approval
-        if self.status == ServiceStatus.OPENING and not should_wait:
+        if self.status == ServiceStatus.OPENING and should_wait:
             try:
                 node.wait_for_node_to_join()
             except TimeoutError:
                 LOG.error(f"New node {node.node_id} failed to join the network")
                 raise
             node.network_state = NodeNetworkState.joined
 
-    def _wait_for_node_to_exist_in_store(self, remote_node, node_id, timeout=3):
+    def _wait_for_node_to_exist_in_store(
+        self, remote_node, node_id, node_status=None, timeout=3
+    ):
         exists = False
         for _ in range(timeout):
-            if self.check_node_exists(remote_node, node_id):
+            if self._check_node_exists(remote_node, node_id, node_status):
                 exists = True
                 break
             time.sleep(1)
         if not exists:
-            raise TimeoutError(f"Node {node_id} has not yet been recorded in the store")
+            raise TimeoutError(
+                f"Node {node_id} has not yet been recorded in the store "
```

Comment:
The most minor of my minor complaints: Can you move the space from the end of the common line, and add it to the start of the optional part (ie - `+ (f" with`)?

Reply:
https://black.readthedocs.io/en/stable/